### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by ghost**
The first commit is just adding x86_64 support for building, but I have some questions on the second one.

Is adding rpms to the repo frowned upon? I couldn't find the answer when I looked.
I added it now since it allows us to just call docker-compose after cloning and get the containers up and running.

We can also discuss if the plugin build files belong here or if we should leave it out for users to handle.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ JAVA=/usr/bin/java
 CLASSPATH=./classes
 VER=1.1
 REL=5
-ARCH=pp64le
+ARCH=$(shell uname -p)
 PROD=ibm-crassd
 NAME=$(PROD)-$(VER)-$(REL).$(ARCH)
```
Comment (thalerj) [inline Makefile]: I suggest adding the RHEL version here as well. The spec files will need to call out the different requirements for RHEL 7 requirements differently due to the change in package naming conventions.

**Review by thalerj — CHANGES_REQUESTED** [review]
I'd recommend removing the binary for openbmctool and instead recommend cloning the repo and generating it during the build process for the crassd rpm. There's a commit out for review for openbmctool to provide a makefile very similar to this one for generating an rpm or deb package binary. Here's a link to that for reference:  https://gerrit.openbmc-project.xyz/c/openbmc/openbmc-tools/+/29184

File: Makefile (hunk 1 above)
Comment (ghost) [reply Makefile]: Added the rhel version as part of the Release after reading https://docs.fedoraproject.org/en-US/packaging-guidelines/DistTag/ and other similar suggestions. More info in the new commits

**Review by thalerj — APPROVED** [review]
I agree with these changes and this looks good.

**Pull request merged**